lib/addons/p5.dom.js

```diff
@@ -735,17 +740,12 @@
       }
       return opt;
     };
-    self.selected = function() {
+    self.selected = function(value) {
```

Comment:
@lmccart does this look right? i tried to add a `validateParameters` but ran into issues since they are not documented independently.

Reply:
ah yes i see the issue with validateParameters. i think it's fine to skip it in this case, at least for now.